### PR TITLE
MIB relocated to Ceph repo

### PR DESCRIPTION
**Pull request by pcuzner**
The MIB has been relocated to the main Ceph repo, so
the MIB file and docs needed to be updated to reflect
this change.

Signed-off-by: Paul Cuzner <pcuzner@redhat.com>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
https://github.com/ceph/ceph/pull/43274

**Comment by votdev** [comment]
I would like to keep all files as they are in this repository, but i'm fine with adding the `SNMP Integration` section in the README.

**Review by votdev — CHANGES_REQUESTED** [review]
I would like to keep all files that are removed by this PR, but i'm fine with adding the SNMP Integration section in the README.

File: README.md [hunk 1]
```diff
@@ -117,31 +117,14 @@ host: promalertmgr.foo.com
 port: 9101
 ```
 
-# SNMP schema
```
Comment (votdev) [inline README.md:120]: Please keep that section.

File: PROMETHEUS-ALERT-CEPH-MIB.txt [hunk 2]
```diff
@@ -1,377 +0,0 @@
-PROMETHEUS-ALERT-MIB DEFINITIONS ::= BEGIN
```
Comment (votdev) [inline PROMETHEUS-ALERT-CEPH-MIB.txt:1]: Please keep that file

**Comment by pcuzner** [comment]
@votdev I'm a bit confused with keeping the MIB in two places. The MIB you've created is using the ceph oid, so shouldn't it exist in only one place? And if so, why would you want to reference information here that could fall out of date relating to the schema?

**Comment by pcuzner** [comment]
@votdev I'll close this PR. I was trying to rationalise how the ceph MIB is consumed across the projects - but I can see now that that doesn't make sense here. Apologies.

Your MIB  has now been merged into ceph, and I have an update to the alerts pending. All of which means the ceph MIb will be receiving a lot of changes in the near future.

**Comment by votdev** [comment]
Sorry for answering so late, was fully occupied by the Bubbles project. I also had my concerns about that because the `prometheus-webhook-snmp` project is a plugin for `Prometheus` and is only indirectly related to `Ceph`.

**Pull request closed without merging**